Yoast/javascript
# Add Turkish Stemmer

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds Turkish stemmer.
* [Yoast SEO Premium] Introduces word form support for Turkish.

## Relevant technical choices:

* The external stemmer is a compiled JavaScript version of a stemmer originally written in Java. This makes the code non easily human-readable. To separate the code from the other code in the yoastseo library, it has been put into a `vendor` folder outside the `src` folder. Relevant build scripts were adjusted.  

## Test instructions

This PR can be tested by following these steps:

In Yoast SEO Premium:

- Add a Turkish text of at least 150 words.
- Set `adayından` as keyphrase and enter `adayın` in the text/meta description/title. The relevant assessments should recognize `adayın` as an instance of the keyphrase. You should see the following assessment feedback:
  - `Keyphrase in meta description: Keyphrase or synonym appear in the meta description. Well done!`
  - `Keyphrase density: The focus keyphrase was found 1 time. That's less than the recommended minimum of 3 times for a text of this length. Focus on your keyphrase! (Number depending on how often you added the keyphrase in the text)`
  - `Keyphrase in title: Does not contain the exact match. Try to write the exact match of your keyphrase in the SEO title and put it at the beginning of the title.`
- Switch the keyphrase and the form in the text/meta description/title around and make sure the assessment results are still
the same.
- Repeat the same process with `edeceğim` as keyphrase and `edeceğimiz` in the text/meta description/title.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-358
